File: Data Structures & Algorithms/concatenation-of-array/submission-0.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        int n = nums.size();
        vector<int> ans = {};
        for(int i = 0; i < 2 * nums.size(); i++) {
            if(i >= nums.size()) {
                ans.push_back(nums[i-n]);
            } else {
                ans.push_back(nums[i]);
            }
        }
        return ans;
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        remainder_map = {}
        for i in range(len(nums)):
            rem = target - nums[i]
            if nums[i] in remainder_map:
                key = nums[i]
                return [remainder_map[key], i];
            else:
                remainder_map[rem] = i
            print(remainder_map)
        

            